src/software/thunderscope/common/proto_parameter_tree_util.py

```diff
@@ -1,7 +1,7 @@
 from pyqtgraph.Qt.QtWidgets import *
 from pyqtgraph import parametertree
-from google.protobuf.json_format import MessageToDict
 from thefuzz import fuzz
+from proto.import_all_protos import *
 
 
 def __create_int_parameter_writable(key, value, descriptor):
```

Comment:
it would be nice to add type annotation

`def __create_int_parameter_writable(key, value, descriptor):` to `def __create_int_parameter_writable(key:str, value:int, descriptor:FieldDescriptor):`

Also, `FieldDescriptor` could be found here: `from google.protobuf.descriptor import FieldDescriptor`